cogs/send_introduction_reminders.py

```diff
@@ -135,12 +134,15 @@ async def send_introduction_reminders(self) -> None:
                     bool(message.components)
                     and isinstance(message.components[0], discord.ActionRow)
                     and isinstance(message.components[0].children[0], discord.Button)
-                    and message.components[0].children[0].custom_id == "opt_out_introduction_reminders_button"  # noqa: E501
+                    and message.components[0].children[0].custom_id
+                    == "opt_out_introduction_reminders_button"
                 )
                 if message_contains_opt_in_out_button:
                     await message.edit(view=None)
 
-            if member not in guild.members:  # HACK: Caching errors can cause the member to no longer be part of the guild at this point, so this check must be performed before sending that member a message # noqa: FIX004
+            if (
+                member not in guild.members
+            ):  # HACK: Caching errors can cause the member to no longer be part of the guild at this point, so this check must be performed before sending that member a message # noqa: FIX004
```

Comment:
Ew Ew Ew. Comments should not be accounted when checking whether a line extends over the maximum line length (as well as `if` statements not being split onto multiple lines)